libraries/AP_RangeFinder/AP_RangeFinder_DroneCAN.cpp

```diff
@@ -81,7 +81,14 @@ AP_RangeFinder_DroneCAN* AP_RangeFinder_DroneCAN::get_dronecan_backend(AP_DroneC
 void AP_RangeFinder_DroneCAN::update()
 {
     WITH_SEMAPHORE(_sem);
-    if ((AP_HAL::millis() - _last_reading_ms) > 500) {
+    if (_status != RangeFinder::Status::Good) {
+        //handle additional states received by measurement handler
+        if (_status == RangeFinder::Status::OutOfRangeLow) {
+            state.distance_m = 0;
```

Comment:
please don't force zero, the actual value is useful in logs

Reply:
Well all the bear traps are in readRangeFinder() as you know. If we really want this for logging I think we should have a last_distance_read() and return nan() if it never read a distance and then use that for logging.